docs/learn/encyclopedia/transactions-specialized/channel-accounts.mdx

```diff
@@ -64,16 +64,16 @@ secret_key = "SDY5TRQSEUSHS7UX26QMNMZ4X543UWZQPJZ7LQQUA3NAKDFVYTAWAB74"
 source_keypair = Keypair.from_secret(secret_key)
 source_account = server.load_account(source_keypair.public_key)
 
-transaction_1 = TransactionBuilder(
+transaction1 = TransactionBuilder(
   source_account = source_account,
   network_passphrase = Network.TESTNET_NETWORK_PASSPHRASE,
-  base_fee = 100
+  base_fee = 640
```

Comment:
Why change to 640, base fee is 100 outside of surge pricing. Maybe add a comment here

Reply:
Going to mark this and next resolved, but by all means reopen if anything unclear about the same-ledger [note](https://github.com/stellar/stellar-docs/blob/d52120537605cc049cf9f3c0270aa21ad8e1520f/docs/learn/encyclopedia/transactions-specialized/channel-accounts.mdx?plain=1#L287) 🔍